src/utils/planet_reader.py

```diff
@@ -48,31 +52,58 @@ def __init__(
 
         :return: None
         """
```

Comment:
Please take a look closely at the changes for planet_reader.py, at this region, b/c I was not so familiar with your changes but there were some merge conflicts here, so I decided to just merge yours in (code is running fine w/ planetdaily).